src/InnloggingBoundary/Innloggingslinje.less

```diff
@@ -6,18 +6,78 @@
     padding: 0.5rem;
     display: flex;
     align-items: center;
+    flex-direction: row;
     justify-content: space-between;
 
+    &__tilbake {
+        margin-right: 1rem;
+        a {
+            text-decoration: none;
+        }
+    }
+
+    .knapp--standard {
+        border-radius: 8px;
+    }
+
     &__identifikator {
         color: white;
         text-decoration: none;
     }
 
+    &__tilbakeknapp {
+        background-color: @navLysGra;
+
+        &--tilbake {
+            display: flex;
+            align-items: center;
+        }
+
+        &--tilbake .chevronboks {
+            padding-right: 0.25rem;
+        }
+
+        &:hover {
+            .nav-frontend-chevron {
+                right: 6px;
+            }
+        }
+    }
+
+    &__identitet {
+        padding-right: 4rem;
+    }
+
     &__loggutknapp {
         background-color: @navLysGra;
     }
 
     @media print {
         display: none;
     }
+
+    .ekspanderbartPanel {
+        background-color: #3e3832;
+        width: 100%;
+
+        &__hode {
+            background-color: #3e3832;
+            color: white;
+        }
+
+        &__heading {
+            color: white;
+        }
+    }
+
+    p.typo-ingress.innloggingslinje__lenkeknapp {
```

Comment:
Hvorfor style på `p`?

Reply:
wrapper godt så ikke css treffer annet en det jeg ønsker.

Reply:
Men trengs den? Styling på tags er generelt no-go etter det jeg hører - jeg synes man burde eksplisitt måtte sette et klassenavn for å få styling.

Reply:
vel setter ikke styling på paragraf generelt men på paragraf som har typo-ingress som child som igjen har innloggingslinjen. Kan sjekke om den treffer uten p først.

Reply:
da var det fikset 💃